frontend/src/main.js

```diff
@@ -3,10 +3,71 @@ import * as THREE from 'three';
 import { FontLoader } from 'three/addons/loaders/FontLoader.js';
 import { TextGeometry } from 'three/addons/geometries/TextGeometry.js';
 
+
```

Comment:
good change, but some of these should be changeable, maybe in the gamestart parameters